### PR TITLE
[LAND-SEE] - Add stories/feature templates, recommended with responsive carousel

### PR DESCRIPTION
**Pull request by afilbert**
PR stubs in stories and feature story (Kim Myeong) template pages & json data, and adds recommended carousel to feature template. Staging URL:

https://landseeampstart.firebaseapp.com/templates/land-see/land-see.amp

recommended.json takes the form of popular.json.

CSS_SIZES all changed again.

### CODE REVIEW

File: templates/land-see/kim-myeong.amp.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <amp-animation id="recommendedSlideIn" layout="nodisplay">
+      <script type="application/json">
+        {
+          "duration": ".4s",
+          "easing": "ease-in",
+          "fill": "both",
+          "direction": "alternate",
+          "animations": [
+            {
+              "selector": ".recommended",
```
Comment (camelburrito) [inline templates/land-see/kim-myeong.amp.html]: namespace?

File: templates/land-see/stories.amp.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "background-animation": {
+    "svg-image": [
+      {
+        "classes": "land-see-background-image-left",
+        "svg": "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 690 551' overflow='visible'><path fill='#ADD8BF' d='M345.048 0l-.032.051-.033-.051L0 550.606h690.031z'/></svg>"
```
Comment (camelburrito) [inline templates/land-see/stories.amp.json]: nit: lowercase for hex values

File: templates/land-see/kim-myeong.amp.html (hunk 1 above)
Comment (afilbert) [reply templates/land-see/kim-myeong.amp.html]: I'm using that class solely for animation selection, like `.popular` and `.recent` in the home template animations. Should I namespace all of them, for consistency? My reasoning keeping it simple is that I would namespace them if I actually needed to define them as classes within `page.css`.

Comment (camelburrito) [reply templates/land-see/kim-myeong.amp.html]: i think it is useful to namespace

**Pull request merged**